src/dds/rs-dds-auto-calibration.h

```diff
@@ -0,0 +1,49 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <src/auto-calibrated-device.h>
+#include <src/ds/d500/d500-auto-calibration.h>
+//#include <src/ds/d500/d500-auto-calibration-handler.h>
+
+
+namespace librealsense
+{
+    class dds_auto_calibrated : public auto_calibrated_interface
```

Comment:
Same in this file -- use clang-format please

Reply:
ok